### PR TITLE
[WIP] Add DISA STIG Compliance

### PR DESCRIPTION
**Pull request by willtome**
This PR transitions to using the DISA-provided roles [from here](https://public.cyber.mil/stigs/supplemental-automation-content/) for STIG compliance.

### CODE REVIEW

File: linux/compliance.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@
 
     - name: Run Compliance Profile
       include_role:
-        name: "redhatofficial.rhel{{ ansible_distribution_major_version }}_{{ compliance_profile }}"
\ No newline at end of file
+        name: "dmeo.compliance.rhel{{ ansible_distribution_major_version }}STIG"
```
Comment (termlen0) [inline linux/compliance.yml]: Typo needs fixing. `dmeo` to `demo`

**Comment by larmbuster** [comment]
I was able to successfully use the LINUX / DISA STIG

**Comment by willtome** [comment]
@larmbuster Looks like we are missing collections for network compliance.

**Review by larmbuster — APPROVED** [review]
Approved assuming adding the ansible.netcommon collection fixed the cisco.ios dependency issue.

**Comment by willtome** [comment]
Closing because this is superseded by https://github.com/ansible/product-demos/pull/61

**Pull request closed without merging**